src/v2/guide/plugins.md

```diff
@@ -6,15 +6,15 @@ order: 304
 
 Plugins usually add global-level functionality to Vue. There is no strictly defined scope for a plugin - there are typically several types of plugins:
 
-1. Add some global methods or properties. e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)
+1. Add some global methods or properties e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)
 
-2. Add one or more global assets: directives/filters/transitions etc. e.g. [vue-touch](https://github.com/vuejs/vue-touch)
+2. Add one or more global assets: directives/filters/transitions etc e.g. [vue-touch](https://github.com/vuejs/vue-touch)
 
-3. Add some component options by global mixin. e.g. [vue-router](https://github.com/vuejs/vue-router)
+3. Add some component options by global mixin e.g. [vue-router](https://github.com/vuejs/vue-router)
 
-4. Add some Vue instance methods by attaching them to Vue.prototype.
+4. Add some Vue instance methods by attaching them to `Vue.prototype`.
```

Comment:
If we're going to remove periods from all of the list items, then we should remove it here as well

```
4. Add some Vue instance methods by attaching them to `Vue.prototype`
```